packages/ragbits-core/src/ragbits/core/llms/clients/base.py

```diff
@@ -12,9 +12,9 @@
 
 
 @dataclass
-class LLMOptions(ABC):
```

Comment:
Why is it no longer an abstract class?

Reply:
Matter of preference, here we simply don't want the user to create an instance of `LLMOptions`, cuz we only support specific implementations for implemented clients.

Reply:
I agree with @micpst - for my marking a class as abstract is not only about being able to mark methods as abstract, but even more importantly about communicating that the class is not intended to be used directly

Reply:
Ok, that makes sense. Initially, in this example, I didn't realize that this class isn't meant to be used directly. I thought it could be used in baseline/no-options clients. So, I assume we're expecting that all clients will have some configuration options, and even if not, the user should subclass this class, even without adding options.